checkout_handler_test.go

```diff
@@ -0,0 +1,154 @@
+package openldap
```

Comment:
This code is copied from [vault-plugin-secrets-ad](https://github.com/hashicorp/vault-plugin-secrets-ad). There are minor modifications to the setup.